renderer/components/preferences/item/index.js

```diff
@@ -75,6 +76,7 @@ class Item extends React.Component {
             display: flex;
             max-width: 100%;
             padding: ${onClick ? '16px' : '32px'} 16px;
+            margin-bottom: ${last ? '16px' : '0'};
```

Comment:
Why not just do this with pure CSS?

```css
.container:last-child {
	margin-bottom: 16px;
}
```

Reply:
This was to push that github button up a bit. This class is used all over settings, so I could do it like that but I would still have to add a class so it can be toggled

Reply:
Ok. I missed the context. It's fine then.